packages/optimizer/lib/transformers/ApplyLayout.js

```diff
@@ -121,10 +121,13 @@ function maybeAddSizerInto(node, layout, width, height) {
   }
 }
 
-function createResponsiveSizer() {
+function createResponsiveSizer(node, width, height) {
+  const padding = (height.numeral / width.numeral) * 100;
+  const style = !node.attribs.heights ? `;padding-top:${parseFloat(padding.toFixed(4))}%` : '';
```

Comment:
Since we're already importing the `hasAttribute` function, might as well use it:

```suggestion
  const style = !hasAttribute(node, 'heights') ? `;padding-top:${parseFloat(padding.toFixed(4))}%` : '';
```